docs/developerguide.rst

```diff
@@ -9,4 +9,60 @@ understand REANA source code and contribute to it.
 Debugging
 ---------
 
-We use ``wdb``.
+In order to debug a REANA component, you first have to install REANA cluster in
+the development mode (see
+`reana-cluster documentation <http://reana-cluster.readthedocs.io/en/latest/developerguide.html#deploying-latest-master-branch-versions>`_).
+Once you have done this, you have to build the image of the component you are
+working on in development mode and we restart the corresponding pod:
+
+.. code-block:: console
+
+   ~/reana-server $ # if we are not connected to the minikube docker daemon \
```

Comment:
Instead of the complex prompt (that different people set up differently):

```
~/reana-server $ foo
~/reana-server $ bar
```

please use `cd` and a simple example prompt, as usual:

```
$ cd src/reana-server
$ foo
$ bar
```